README.md

```diff
@@ -1 +1,196 @@
+
+## Conference Structure
+- Platform: Hopin
+- Communication: Zulip Chat
+- 5 tracks on stage
+    * 3 general tracks
+    * Diversity
+    * Extended
+- Multiple Sessions
+- Community Booths
+- Bofs
+
+## Working Groups
+
+* Conference Chair: Chandan Kumar
+* Conference Co-Chairs:
+    - Kumar Anirudha
+    - Ritesh Agarwal
+* Emritus Chair: Sayan Chowdhury
+
+### Guidelines
+
+* One person can volunteer in multiple working group, can lead only on working group.
+
+### Accessibility
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Faccessibility)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Volunteers
+
+- [Github Issues]()
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### CFP (Call For Proposals)/Keynotes
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fcfp)
+- Meeting:
+- Chair: Sukanya
+- Co-chair:
+- Shadow:
+- Members
+
+### Content and Design
+
+- [Github Issues](https://github.com/pythonindia/inpycon-blog/issues/)
+- Meeting:
+- Chair: Vipul Gupta
+- Co-chair:
+- Shadow:
+- Members
+  * Nikhil Maan
+  * Arvind
+
+### Dev Sprint/Workshops
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fdevsprint)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Mentorship/Editorial
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fmentorship)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Platform
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fplatform)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Review
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Freview)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+
+### Bofs
+
+- [Github Issues]()
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Outreach and Communication (social media, community partners)
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fsocial-media)
+- Meeting:
+- Chair: Abhishek
+- Co-chair:
+- Shadow:
+- Members
+
+### Technology
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Ftechnology)
+- Meeting:
+- Chair: Nabarun
+- Co-chair:
+- Shadow:
+- Members
+
+### Finance
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fbilling)
+- Meeting:
+- Chair:
+- Co-chair:
+- Shadow:
+- Members
+
+### Diversity
+
+- [Github Issues](https://github.com/pythonindia/inpycon2021-tasks/issues?q=is%3Aopen+is%3Aissue+label%3Awg%2Fdiversity)
+- Meeting:
+- Chair: Sukanya
+- Co-chair:
+- Shadow:
+- Members
+
+### Tickets
```

Comment:
Since, this year FInance team will be mostly be looking into Tickets. Thoughts of clubbing them?